src/proto/api_messages.js

```diff
@@ -22,7 +22,7 @@
 /**
  * @interface
  */
-class Message {
+ class Message {
```

Comment:
Is this extra space necessary?

Reply:
No, removed. Not in the original file, which this should be copy-pasted from, so I'm not sure how it sneaked in.